arviz/rcparams.py

```diff
@@ -240,7 +240,7 @@ def validate_iterable(value):
     ),
     "plot.matplotlib.constrained_layout": (True, _validate_boolean),
     "plot.matplotlib.show": (False, _validate_boolean),
-    "stats.credible_interval": (0.94, _validate_probability),
+    "stats.hpd_interval": (0.94, _validate_probability),
```

Comment:
I don't really know if anybody else apart from me is using rcparams, so I am not really opposed to just changing them right away, however, if possible it would be a good idea to have a warning if an rcfile has `stats.credible_interval` telling them to update. I could add some `deprecated` list for rcparams or something similar.

Reply:
Oh this is a good point. I will add

Also I use them btw! They are great

Reply:
I base my assumption on the fact that no issue about bad rcparams behaviour has been opened and I find it hard to believe there are actually no issues at all :thinking: